exonum/benches/block.rs

```diff
@@ -76,25 +74,25 @@ mod tests {
             height: u64,
             txs: &[Hash],
             pool: &BTreeMap<Hash, Box<Transaction>>,
-        ) -> Patch {
+        ) -> Arc<View> {
             blockchain
-                .create_patch(ValidatorId::zero(), Height(height), txs, pool)
+                .create_block(ValidatorId::zero(), Height(height), txs, pool)
                 .1
         }
 
         for i in 0..100 {
             let (txs, pool) = prepare_txs(i, 1000);
-            let patch = execute_block(&blockchain, i, &txs, &pool);
-            blockchain.merge(patch).unwrap();
+            let _fork = execute_block(&blockchain, i, &txs, &pool);
+            //  fork.commit();
+            //  blockchain.merge(patch).unwrap();
```

Comment:
Will `commit` still be needed here? If not, I propose to remove the commented code:
```rust
let _ = execute_block(&blockchain, i, &txs, &pool);
```